hubblestack/extmods/fileserver/azurefs.py

```diff
@@ -201,9 +201,19 @@ def update():
         except Exception as exc:
             log.exception('Error occurred fetching blob list for azurefs')
 
-            if not __opts__['delete_inaccessible_azure_containers'] or not "<class 'azure.common.AzureHttpError'>" in str(type(exc)) :
+            if not __opts__['delete_inaccessible_azure_containers'] \
+               or ( not "<class 'azure.common.AzureHttpError'>" in str(type(exc)) \
+                    and \
+                    not "<class 'azure.common.AzureMissingResourceHttpError'>" in str(type(exc))
```

Comment:
If we delete a container, it falls under 'azure.common.AzureMissingResourceHttpError' error.